docs/src/administration/sso/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Single sign-on (SSO)"
+weight: 4
+description: |
+  Platform.sh allows you to set up mandatory SSO with a third-party identity provider (IdP) for all your users.
+sidebarTitle: "Single sign-on (SSO)"
```

Comment:
You don't need to include `sidebarTitle` on pages where the sidebar is the same as `title`